packages/blade/src/components/Radio/__tests__/RadioGroup.web.test.tsx

```diff
@@ -20,7 +20,7 @@ describe('<RadioGroup />', () => {
       </RadioGroup>,
     );
     expect(container).toMatchSnapshot();
-    expect(getByRole('group')).toHaveTextContent(labelText);
+    expect(getByRole('radiogroup')).toHaveTextContent(labelText);
```

Comment:
web pe to we are setting `group` only right? so why are we checking `radiogroup` here?

Reply:
>web pe to we are setting group only right? so why are we checking radiogroup here?

RadioGroup should be `radiogroup` & CheckboxGroup should be `group` on both web and native. 

>also the comment says that it shall be radiogroup only for both?

The comment is only for the RadioGroup component.

@anuraghazra Can you confirm?

Reply:
Yes it should be [group only](https://russmaxdesign.github.io/accessible-forms/fieldset-checkbox-group-role-group.html), there is nothing called a "checkboxgroup" and using radiogroup for checkbox will be misleading for screen reader users.